src/libraries/Microsoft.Extensions.Options/gen/Emitter.cs

```diff
@@ -374,12 +374,83 @@ public void EmitCompareAttribute(string modifier, string prefix, string classNam
 """);
         }
 
-        public void EmitRangeAttribute(string modifier, string prefix, string className, string suffix)
+        public void EmitRangeAttribute(string modifier, string prefix, string className, string suffix, bool emitTimeSpanSupport)
         {
             OutGeneratedCodeAttribute();
 
             string qualifiedClassName = $"{prefix}{suffix}_{className}";
 
+            string initializationString = emitTimeSpanSupport ?
+            """
+                                if (OperandType == typeof(global::System.TimeSpan))
+                                {
+                                    if (!global::System.TimeSpan.TryParse((string)Minimum, culture, out global::System.TimeSpan timeSpanMinimum) ||
```

Comment:
Could this parsing happen at compile time, or do we have to honor local culture settings?

Reply:
The culture is one reason. The other reason is if I parse it at compile time will need to pass the parsed values to the attribute constructors (possibly as objects) which will need adding a new constructor there and will need to add more emitting code to generate such constructor parameter. I don't think this is worth doing it in servicing release.